sdk/js/package.json

```diff
@@ -87,5 +84,10 @@
     "elliptic": "^6.5.4",
     "js-base64": "^3.6.1",
     "near-api-js": "^1.0.0"
+  },
+  "optionalDependencies": {
```

Comment:
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#optionaldependencies
>build failures do not cause installation to fail.

Since injective doesn't build on local imports not in a sibling directory, ignore the dep if it fails. The injective sdk is only used in a couple places that interact with injective, so this should not affect evm only generic relaying any time soon